packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -36,6 +36,7 @@ import { SharedTreeProviders } from "../trees/shared/SharedTreeProviders";
 import { ZoweExplorerExtender } from "../extending/ZoweExplorerExtender";
 import { FilterDescriptor, FilterItem } from "../management/FilterManagement";
 import { AuthUtils } from "../utils/AuthUtils";
+import { DeferredPromise } from "@zowe/imperative";
```

Comment:
should we use the `imperative.` version from the zowe-explorer-api package here?